README.md

```diff
@@ -21,6 +21,9 @@ This module has been tested to work on the following systems with Puppet v3.x on
  * Solaris 10
  * Ubuntu 10.04 LTS (Lucid Lynx)
  * Ubuntu 12.04 LTS (Precise Pangolin)
+ * OpenBSD 5.4
```

Comment:
I checked the man pages which date back to OpenBSD 3 and all versions 3-5 support this option. Suggest Listing
```
* OpenBSD 3
* OpenBSD 4
* OpenBSD 5
```